tests/fixtures/rendering.yml

```diff
@@ -250,31 +250,42 @@
     '#id': file_link
     '#fields':
       files:
-        - title: "Example document 1"
-          mime: "application/pdf"
-          url: "http://example.com/document1.pdf"
+        - title: "Example document"
```

Comment:
Once you add the lowercase part above, add a test with an uppercase extension too.